inapppy/errors.py

```diff
@@ -2,7 +2,8 @@
 class InAppPyValidationError(Exception):
     """ Base class for all validation errors """
     raw_response = None
+    error_message = None
 
-    def __init__(self, raw_response=None):
+    def __init__(self, error_message=None):
         super(InAppPyValidationError, self).__init__()
-        self.raw_response = raw_response
```

Comment:
I would leave `raw_response` as is, just add `error_message` as additional parameter.